frontend/actions/documentActions.js

```diff
@@ -124,6 +125,56 @@ export function registerUserLeavingDocument ({
   }
 }
 
+export function uploadMediaToBucket ({
+  api,
+  bucket,
+  files
+}) {
+  return (dispatch, getState) => {
+    dispatch(
+      setRemoteDocumentStatus(Constants.STATUS_SAVING)
+    )
+
+    Promise.all(
+      Array.from(files).map(
```

Comment:
Could you make the function argument part of the same line as the function name to avoid an unnecessary level of indentation?